docs/deployment/required/cert-manager.md

```diff
@@ -29,7 +29,6 @@ kubectl get certificates
 ## Configure Helm Values
 Create a new helm values file for cert-manager with the following command:
 ```bash
-mkdir -p deployments/cert-manager
```

Comment:
Shouldn't the first command stay there?

Reply:
Its not needed because the first curl commands creates the folder with `--create-dirs`